README.md

```diff
@@ -56,6 +56,7 @@ An integration test is prepared for contracts and federators. To properly run in
 1. Check `infura.key` in `bridge`
 1. Check `federator.key` in `federator/config`
 1. Check `config.js` in `federator/config`
+1. Check your `networkName` in `bridge/migrations/4_deploy_erc1820.js`
```

Comment:
Add something like  "that this step is necessary only if the network you are working on does not have ERC1820 deployed"  and i'll merge it

Reply:
Okay.

Reply:
Done. Please help to review this PR.